package.json

```diff
@@ -22,7 +22,7 @@
 		"dev": "node scripts/watch.js",
 		"generate": "graphql-codegen",
 		"lc": "jsgl --local ./",
-		"lint": "eslint \"assets/src/**\" --config \".eslintrc.json\" --ext .js,.jsx,.ts,.tsx",
+		"lint": "./node_modules/.bin/eslint \"assets/src/**\" --config \".eslintrc.json\" --ext .js,.jsx,.ts,.tsx",
```

Comment:
nice, since we don't have a script to install globally eslint on the server :+1: